testpilot/frontend/static-src/styles/modules/_main-header.scss

```diff
@@ -7,6 +7,8 @@
     font-weight: 700;
     margin: 10px 0;
 
+    a { cursor: pointer; }
```

Comment:
@ktbee I'm surprised this was needed. Does the link not get the pointer style without this?